spark/v2.4/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/ArrowVectorAccessorFactory.java

```diff
@@ -78,6 +79,17 @@ public UTF8String ofRow(VarCharVector vector, int rowId) {
     public UTF8String ofBytes(byte[] bytes) {
       return UTF8String.fromBytes(bytes);
     }
+
+    @Override
+    public UTF8String ofByteBuffer(ByteBuffer byteBuffer) {
+      if (byteBuffer.hasArray()) {
+        return UTF8String.fromBytes(
+                byteBuffer.array(), byteBuffer.arrayOffset() + byteBuffer.position(), byteBuffer.remaining());
```

Comment:
Nit: over-indented (should be 4 spaces from the start of `return` on the line above).

Reply:
Thanks, I fixed these. Checkstyle didn't seem to mind...

Reply:
Thanks for letting me know. I’ll see if I can add a checkstyle rule for that or update one to catch it!